subprojects/language-groovy/language-groovy.gradle

```diff
@@ -2,11 +2,19 @@
 // Compiler daemon
 sourceCompatibility = 1.6
 
+repositories {
+    maven {
+        url  "https://dl.bintray.com/incap/incap"
+    }
+}
+
 dependencies {
     compile project(":core")
     compile project(":platformJvm")
     compile project(":languageJava")
 
+    compile 'org.gradle.incap:incap-build-client:0.0.2'
```

Comment:
why not 0.0.4 ?

Reply:
Argh, this is the wrong code.  I'll fix it.